pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/ReadableSequentialDataTest.java

```diff
@@ -11,6 +20,38 @@ protected ReadableSequentialData emptySequence() {
         return new StubbedSequence(new byte[0]);
     }
 
+    @NonNull
+    private ReadableSequentialData throwingSequence() {
+        return new StubbedSequence(new byte[] { 1 }, () -> new EOFException());
+    }
+
+    @Test
```

Comment:
These new test cases can be moved to `ReadableSequentialTestBase`, too

Reply:
Same reason as above: I didn't find an easy way to reach a branch that would trigger the exception in one of the descendent classes.